src/screens/Login/index.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { Component } from "react";
+import {
+  FormGroup,
+  FormControl,
+  ControlLabel
+} from "react-bootstrap";
+import { Auth } from 'aws-amplify';
+import LoaderButton from '../../components/LoaderButton';
+import "./index.css";
+
+class Login extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      isLoading: false,
+      email: "",
+      password: "",
+    };
+  }
+
+  validateForm() {
+    return (
+      this.state.email.length > 0 &&
+      this.state.password.length > 0
+    );
```

Comment:
Nitpicking: we can shorten the code a little bit
```suggestion
    const { email, password } = this.state;
    return email && password;
```

Reply:
Thank Huy, I'll update it.